src/internal/menu/StartMenu.tsx

```diff
@@ -127,31 +145,37 @@ export function StartMenuComponent({header, mode, open}: {header: AppHeader, mod
     }
 
     currentSessionNav.toggleAttribute('hidden', (activeTab) ? true : false);
-  }, [header, activeTab]);
+  }, [activeTab, tabs]); // TODO: when the current session button is clicked the tabs are null.
```

Comment:
There is an issue here that needs to be fixed see code comment

Reply:
Good point! Thanks for mentioning this point. I've split this `useEffect` into two `useEffect`s, one for the initalization and one for the show/hide functionality. This should also make the code more readabe.